pkg/doc/signature/suite/ecdsasecp256k1signature2019/public_key_verifier.go

```diff
@@ -7,97 +7,13 @@ SPDX-License-Identifier: Apache-2.0
 package ecdsasecp256k1signature2019
 
 import (
-	"crypto"
-	"crypto/ecdsa"
-	"errors"
-	"fmt"
-	"math/big"
-
-	"github.com/btcsuite/btcd/btcec"
-
-	sigverifier "github.com/hyperledger/aries-framework-go/pkg/doc/signature/verifier"
-)
-
-const (
-	secp256k1KeySize = 32
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/verifier"
 )
 
-// PublicKeyVerifier verifies a secp256k1 signature taking public key bytes and JSON Web Key as input.
-// NOTE: this verifier is present for backward compatibility reasons and can be removed later.
-// Please use CryptoVerifier or your own verifier.
-type PublicKeyVerifier struct {
+// NewPublicKeyVerifier creates a signature verifier that verifies a ECDSA secp256k1 signature
+// taking Ed25519 public key bytes as input.
+func NewPublicKeyVerifier() *verifier.PublicKeyVerifier {
```

Comment:
This is how a signature verifier for the non-composite signature suite is created.